static_src/skins/cg/index.js

```diff
@@ -64,7 +64,10 @@ export const config = {
     ]
   },
   docs: {
-    cli: 'https://docs.cloud.gov/getting-started/setup/'
+    cli: 'https://docs.cloud.gov/getting-started/setup/',
+    concepts_spaces: 'https://docs.cloud.gov/getting-started/concepts/',
+    deploying_apps: 'https://cloud.gov/docs/apps/deployment/',
+    use: 'https://docs.cloud.gov/intro/overview/using-cloudgov-paas/'
```

Comment:
These should be updated since `docs.cloud.gov` redirects to -> `cloud.gov/docs` now.